src/Microsoft.AspNetCore.Sockets/TransportType.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Microsoft.AspNetCore.Sockets
+{
+    [Flags]
+    public enum TransportType
+    {
+        /// <summary>
+        /// Every transport
+        /// </summary>
+        All = Streaming | LongPolling,
```

Comment:
Yea, this was something that I copied from old new signalr. Wanted to see what people think of the entire approach (enum vs strings etc).